LICENSE.md

```diff
@@ -0,0 +1,6 @@
+# Software License Agreement
+
+**TinyMCE** – <https://github.com/tinymce/tinymce>](https://github.com/tinymce/tinymce)
```

Comment:
missing a `[` at the start of the markdown link

oops, I should've looked at the rendered file 🙃

Reply:
Yeah I just copied in what I got from Fredrik didn't verify it closely. I fixed this pushed straight to master seemed silly to take time from people to review a one character change.